esti/s3_gateway_test.go

```diff
@@ -168,6 +171,25 @@ func TestS3ReadObject(t *testing.T) {
 				t.Errorf("Got contents \"%s\" but expected \"%s\"", string(got), contents)
 			}
 		})
+		t.Run("ExistsPreSigned", func(t *testing.T) {
+			// using presigned URL so we can check the headers
+			// We expect the Content-Length header to be set
+			preSignedUrl, err := client.Presign(ctx, http.MethodGet, repo, goodPath, time.Second*60, url.Values{})
```

Comment:
1. Missed it before - can you rename `preSignedUrl` to `preSignedURL`
2. Didn't want to use `time.Minute`?

Reply:
Since the presigned URL duration is in seconds, I thought it was more appropriate to use a Duration in seconds

Reply:
No. 1 - Done

Reply:
duration is in nanoseconds. and the constant time.Minute is 60 time.second